test/parallel/test-readline-interface.js

```diff
@@ -881,6 +882,56 @@ for (let i = 0; i < 12; i++) {
     rli.close();
   }
 
+  // Calling the promisified question
+  {
+    const [rli] = getInterface({ terminal });
+    const question = util.promisify(rli.question).bind(rli);
+    question('foo?')
+    .then(common.mustCall((answer) => {
+      assert.strictEqual(answer, 'bar');
+    }))
+    .catch(common.mustNotCall());
```

Comment:
Nit: This .catch is probably not required since unhandled rejections crash the process in our tests which will fail the test with a non-zero status code.

Reply:
I have removed it but in my view it is clearer when looking at the test what the expected behavior is than relying on external rejection code. :)